.vsts-pipelines/templates/build-steps.yml

```diff
@@ -3,6 +3,8 @@ phases:
   parameters:
     agentOs: Windows
     beforeBuild:
+      - powershell: "git submodule update --init"
```

Comment:
@natemcmaster is there a better way?

Reply:
Change the checkout settings to include submodules.

Reply:
Where?

Reply:
In the VSTS build definition. Unfortunately this is one you can't set via YAML yet. I've updated it for you

https://dotnet.visualstudio.com/public/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=27

![image](https://user-images.githubusercontent.com/2696087/43410827-192ecace-93dd-11e8-9cd5-0f0685ca6a9a.png)